P3-Environment-Setup/content.md

```diff
@@ -1,68 +1,70 @@
+Next, let's get the latest version of Ruby installed: `rbenv install 2.3.0`.  This will take a few minutes.  Stop and get assistance if you receive any errors.
 
-## Setting up your Rails Project
+We can check which version of Ruby is being used by the system: `ruby -v`.  You should see that `system` is the current selected version of Ruby.  We want to be using 2.3.0, so let's tell rbenv to set the globally used Ruby: `rbenv global 2.3.0`.  Running `ruby -v` should now show 2.3.0 as the selected version.  Stop and get assistance if you receive any errors.
 
-In case you haven't already read the instructions on creating a new Rails project and setting up your gems, this would be a good time to cover that in the RailsTutorial.org
+Lastly, we need to install Bundler, a version manager for Ruby libraries, and Rails:
 
-https://www.railstutorial.org/book/_single-page#sec-installing_rails
+```
+gem install bundler
+gem install rails
+rbenv rehash
+```
+The rehash command is necessary only when installing gems without using Bundler (we'll cover this in more detail later) for the first time in a given version of Ruby (each version of Ruby maintains its own set of gems due to version dependency issues).  
 
-Follow the instructions on setting up a new Rails Project though I would name the project a more descriptive name than hello_project. Perhaps something like reddit_on_rails.
+## Rails
+Rails is an opinionated framework, and comes with a set of generators to make our lives easier.  The first generator we're going to use is `new`, which will create a complete skeleton of a rails project for us.  Only one of the team members needs to perform this step, so please pair for it:
 
-You can stop once you hit section 1.3.3. That should be a basic project at this point. 
+`rails new $PROJECT_NAME` -- replace $PROJECT_NAME with the name used for the repo created under the **Github** section.  Navigate in to your project using the **c**hange **d**irectory command: `cd $PROJECT_NAME`.  Then, using the **l**i**s**t command, check out the directory structure that the generator created for you: `ls -la`.
 
-## Configuring Editors
+Next, we need to create a new git repository to store changes and allow you to share the source code of your new project with your team members.  In the directory of your new rails project, run:
 
-Make sure that you both have editors set to the same preferences. Tabs vs spaces is a surefire way to make sure that you have conflicts when editing the same file. The general preference for ruby projects is to have two **spaces** without any hard tabs. 
+TODO: These are incomplete. DANK
 
-If you're using Sublime it's pretty easy to configure your project to use two spaces.
+```
+git init
+git add .
+git commit -m "Initial commit"
+```
 
-Go to the bottom right Sublime and choose from Plain Text (or whatever it says) to Ruby.
+## Pushing your project to Github
```

Comment:
You're presuming they haven't used git but you mentioned below there will be a git lecture (in the comment about HTTPS vs SSH)?

My suggestion is to do RBENV -> Github -> Pushing Your Project
instead of Github -> RBEnv -> Pushing your project

Might be simpler and builds on top of each other better than Github -> RBEnv -> Pushing Your Project.